### DISCUSSION TITLE
FPS are capped to 60 if the app is maximized or borderless

### DISCUSSION THREAD

**Discussion by SamiraUrzua** [NiiightmareXD/windows-capture#48] [Q&A]
I'm running a game at 120 fps in a 120 fps monitor. When the game is a borderless window mode fps are capped to 60, when I'm in windows mode with the game maximized it runs at 60 fps but if the game isn't maximized and it's on a window with the same resolution it runs at 120 fps.

It  doesn't happen with all apps, for example Notepad doesn't have the same issue, but windows 11 calculator does.

```
from windows_capture import WindowsCapture, Frame, InternalCaptureControl
import time

# Initialize WindowsCapture with the specific window name "VRChat"
capture = WindowsCapture(
    cursor_capture=False,
    draw_border=False,
    monitor_index=None,
    window_name="Calculator",
)

# Initialize variables for FPS calculation
frame_count = 0
start_time = time.time()

# Called every time a new frame is available
@capture.event
def on_frame_arrived(frame: Frame, capture_control: InternalCaptureControl):
    global frame_count, start_time
    # Update frame count
    frame_count += 1
    
    # Calculate and print FPS every second
    if time.time() - start_time >= 1:
        print(f"FPS: {frame_count}")
        frame_count = 0
        start_time = time.time()

# Called when the capture item closes
@capture.event
def on_closed():
    pass

# Start the capture session
capture.start()
```

**Comment by sugphaze** [comment]
I have the same problem.My game FPS is 75 and the real capture fps is limited by 37.5 . I asked ChatGPT and he told me:"Windows Graphics Capture API has a design characteristic: it synchronizes to the DXGI swap chain's presentation interval, and when games use double buffering with vertical sync enabled, capture gets locked to half the refresh rate. This happens because it needs to wait for complete frame presentation before capturing, resulting in only capturing every other frame.“
